docs/iot-gateway/config/request.md

```diff
@@ -0,0 +1,324 @@
+      "httpMethod": "GET",
+      "httpHeaders": {
+        "ACCEPT": "application/json"
+      },
+      "allowRedirects": true,
+      "timeout": 0.5,
+      "scanPeriod": 100,
+      "converter": {
+        "type": "custom",
+        "deviceNameJsonExpression": "SD8500",
+        "deviceTypeJsonExpression": "SD",
+        "extension": "CustomRequestUplinkConverter",
+        "extension-config": [
+          {
+            "key": "Totaliser",
+            "type": "float",
+            "fromByte": 0,
+            "toByte": 4,
+            "byteorder": "big",
+            "signed": true,
+            "multiplier": 1
+          },
+          {
+            "key": "Flow",
+            "type": "int",
+            "fromByte": 4,
+            "toByte": 6,
+            "byteorder": "big",
+            "signed": true,
+            "multiplier": 0.01
+          }
+        ]
+      }
+    }
+  ],
+  "attributeUpdates": [
+      {
+        "httpMethod": "POST",
+        "httpHeaders": {
+          "CONTENT-TYPE": "application/json"
+        },
+        "timeout": 0.5,
+        "tries": 3,
+        "allowRedirects": true,
+        "deviceNameFilter": "SD.*",
+        "attributeFilter": "send_data",
+        "requestUrlExpression": "sensor/${deviceName}/${attributeKey}",
+        "valueExpression": "{\"${attributeKey}\":\"${attributeValue}\"}"
+      }
+  ],
+  "serverSideRpc": [
+    {
+      "deviceNameFilter": ".*",
+      "methodFilter": "echo",
+      "requestUrlExpression": "sensor/${deviceName}/request/${methodName}/${requestId}",
+      "responseTimeout": 1,
+      "httpMethod": "GET",
+      "valueExpression": "${params}",
+      "timeout": 0.5,
+      "tries": 3,
+      "httpHeaders": {
+        "Content-Type": "application/json"
+      }
+    },
+    {
+      "deviceNameFilter": ".*",
+      "methodFilter": "no-reply",
+      "requestUrlExpression": "sensor/${deviceName}/request/${methodName}/${requestId}",
+      "httpMethod": "POST",
+      "valueExpression": "${params}",
+      "httpHeaders": {
+        "Content-Type": "application/json"
+      }
+    }
+  ]
+}
+
+{% endhighlight %}
+
+</details>
+
+
+### General section
+
+| **Parameter** | **Default value**                 | **Description**                                           |
+|:-|:-|-
+| host          | **http://127.0.0.1:5000**         | Domain address or ip of the server.                       |
+| SSLVerify     | **true**                          | Verify or no SSL certificate on the server if available.  |
+|---
+
+
+### Security section
+ 
+{% capture requestconnectorsecuritytogglespec %}
+Basic<small>Recommended</small>%,%username%,%templates/iot-gateway/request-connector-basic-security-config.md%br%
+Anonymous<small>No security</small>%,%anonymous%,%templates/iot-gateway/request-connector-anonymous-security-config.md{% endcapture %}
+
+{% include content-toggle.html content-toggle-id="requestConnectorCredentialsConfig" toggle-spec=requestconnectorsecuritytogglespec %}
+
```

Comment:
You may add some comment here like this: Tell how to implement other security options (point in code) or provide a roadmap of other options.